front/store/user.js

```diff
@@ -0,0 +1,33 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+/**
+ * Default state object with initial values.
+ */
+const initialState = {
+  name: "Quentin",
+  email: "quentin@gmail.com",
+};
+
+/**
+ * Create a slice as a reducer containing actions.
+ *
+ * In this example actions are included in the slice. It is fine and can be
+ * changed based on your needs.
+ */
+const userSlice = createSlice({
+  name: "user",
+  initialState,
+  reducers: {
+    setName: (state, action) => {
```

Comment:
Are you using those ?

Reply:
Same comment as above, its an example reducer and will be overwritten when we integrate the auth